testsuite/smoke-test/src/client.rs

```diff
@@ -8,13 +8,19 @@ use crate::test_utils::{
 };
 use cli::client_proxy::ClientProxy;
 use diem_types::{ledger_info::LedgerInfo, waypoint::Waypoint};
+use diem_config::config::NodeConfig;
 
 #[test]
 fn test_create_mint_transfer_block_metadata() {
     let (env, client) = setup_swarm_and_client_proxy(1, 0);
 
-    // This script does 4 transactions
-    check_create_mint_transfer(client);
+    if NodeConfig::default_for_validator().execution.decoupled {
```

Comment:
we can leave smoke test unchanged for now

Reply:
I changed the smoke test back.